tools/make-framework

```diff
@@ -243,6 +248,21 @@ if [[ -n "$sim_and_device" && -z "$make_xcframework" ]]; then
 	exit 1
 fi
 
+# this one-liner is from https://stackoverflow.com/questions/59895/how-can-i-get-the-source-directory-of-a-bash-script-from-within-the-script-itsel
```

Comment:
Totally unimportant, but:

```suggestion
# this one-liner is from https://stackoverflow.com/questions/59895/how-can-i-get-the-source-directory-of-a-bash-script-from-within-the-script-itself
```

Reply:
Oddly enough, this is correct. I checked the link twice because I didn't believe it.